myapp/views.py

```diff
@@ -49,3 +49,13 @@ def logout_view(request):
 def single_stock_historic(request, symbol):
 	data = stock_api.get_stock_historic_prices(symbol, time_range='1m')
 	return JsonResponse({'data': data})
+
+
+def single_stock_estimates(request, symbol):
+	# Note: in the estimates request I don't get the company name so I got it from

```

Comment:
yes. symbol is better to pass than company name

Reply:
Removed company name.